src/app/registration/facility-select/facility-select.component.spec.ts

```diff
@@ -32,31 +32,33 @@ describe('FacilitySelectComponent', () => {
 
   describe('availability text display', () => {
     function setupVisitTimeState(facilities: any[], visitDate: Date): void {
+      // initialize the component
       component.facilities = facilities;
       component.ngOnInit();
       fixture.detectChanges();
 
-      const facilityElement = fixture.debugElement;
-      const passTypeElement = facilityElement.query(
-        By.css('[data-testid="passtype-select"]')
-      ).nativeElement;
-      passTypeElement.value = passTypeElement.options[1].value;
-      passTypeElement.dispatchEvent(new Event('change'));
-      fixture.detectChanges();
-
+      // set the visit date
       component.myForm.patchValue({
         visitDate: {
           year: visitDate.toLocaleString('en-US', { year: 'numeric', timeZone: 'America/Vancouver' }),
           month: visitDate.toLocaleString('en-US', { month: 'numeric', timeZone: 'America/Vancouver' }),
           day: visitDate.toLocaleString('en-US', { day: 'numeric', timeZone: 'America/Vancouver' }),
         },
       });
-
       const datePickerComponent = fixture.debugElement.query(
         By.css('app-date-picker')
       );
       datePickerComponent.triggerEventHandler('formChangeEvent', null);
       fixture.detectChanges();
+
+      // pick a facility
+      const facilityElement = fixture.debugElement;
+      const passTypeElement = facilityElement.query(
+        By.css('[data-testid="passtype-select"]')
+      ).nativeElement;
```

Comment:
Prefer: 
```
const passTypeElement = facilityElement.query(By.css('[data-testid="passtype-select"]')).nativeElement;
```
120 col is generally acceptable, 80 col wrapping is no longer desired.